src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -93,8 +104,8 @@ struct grpc_tls_credential_reload_config
       gpr_log(GPR_ERROR, "schedule API is nullptr");
       if (arg != nullptr) {
         arg->status = GRPC_SSL_CERTIFICATE_CONFIG_RELOAD_FAIL;
-        arg->error_details =
-            gpr_strdup("schedule API in credential reload config is nullptr");
+        arg->error_details->set_err_details(
```

Comment:
as suggested before, making status part of `error_details` makes this code simpler